apps/vs-code-designer/src/app/commands/workflows/cloudToLocal.ts

```diff
@@ -0,0 +1,95 @@
+// Used cloudToLocal.ts as a template to create this file
+// This file is used to import a Logic App from Azure to the local workspace
+// Still need to implement the rest of the imports
+import * as fs from 'fs';
+import { getAccountCredentials } from '../../utils/credentials';
+import type { TokenCredential } from '@azure/core-auth';
+import { LogicManagementClient } from '@azure/arm-logic';
+import * as path from 'path';
+import AdmZip = require('adm-zip');
+
+async function isStateful(logicApp: any): Promise<boolean> {
+  // Need to check if the Logic App is stateful
+  // This is a placeholder implementation
+  return logicApp.stateful || false;
```

Comment:
NIT: If we `logicApp.stateful` is a boolean, there is no need for the return comparison